tc/tc_util.h

```diff
@@ -66,9 +66,12 @@ const char *get_tc_lib(void);
 struct qdisc_util *get_qdisc_kind(const char *str);
 struct filter_util *get_filter_kind(const char *str);
 
+int read_prop(char *dev, char *prop, long *value);
```

Comment:
This function is already in `ip/iptuntap.c`. Instead of defining it twice, can we move it to a more general file so that both `ip/iptuntap.c` and the TC files can use it?